### PR TITLE
ROCm gpg url changed

### PR DESCRIPTION
**Pull request by emcastillo**
Jenkins, test this please

### CODE REVIEW

File: .pfnci/script.sh [hunk 1]
```diff
@@ -67,7 +67,7 @@ main() {
       apt install python3.7-dev python3-apt python3-pip python3-setuptools -qqy
       python3.7 -m pip install cython numpy
 
-      wget -qO - http://repo.radeon.com/rocm/apt/debian/rocm.gpg.key | apt-key add -
+      wget -qO - http://repo.radeon.com/rocm/apt/debian/dists/xenial/Release.gpg | apt-key add -
```
Comment (leofang) [inline .pfnci/script.sh:70]:
```suggestion
      wget -qO - http://repo.radeon.com/rocm/rocm.gpg.key | apt-key add -
```

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by kmaehashi** [comment]
LGTM! Merging as `.pfnci/script.sh` is obviously not related to Jenkins.

**Pull request merged**